### ISSUE TITLE
Transition to a new route form when the modal is open

### ISSUE THREAD

**Issue by zhe1ka** [pradel/react-responsive-modal#239]
hey, first of all, thanks for simple and a great package, i found it very useful.

i'm facing the same problem which was reported by another person some time ago.
https://github.com/pradel/react-responsive-modal/issues/63

as i understand the issue was just closed and wasn't fixed. is it possible to fix it?
it's very simple to reproduce: call the function which closes a modal and after that go to another route, in my case it's:
`this.props.history.push('/new-route')` 

thanks.

## Environment

| Tech                   | Version |
| ---------------------- | ------- |
| react-responsive-modal |    3.5.1    |
| React                  |    16.6.3    |
| browser                |     *    |

**Comment by pradel** [comment]
Hey, do you have a sample repo that show the issue so I can reproduce it easily?

**Comment by zhe1ka** [comment]
i've just created a repo with an example
https://github.com/zhe1ka/react-responsive-modal

actually the problem is appeared when we use `CSSTransitionGroup` package for a smooth transition between pages. is it possible to fix?

**Comment by pradel** [comment]
Thanks for the reproduction, will take a look this weekend :)

**Comment by zhe1ka** [comment]
any news? :)

**Comment by pradel** [comment]
Hey, didn't had time to take a look this weekend, this next weeks will be pretty packed, don't know when I will have time to get back to it :(
If you feel like debugging and create a pr go for it 🙏

**Comment by zhe1ka** [comment]
pk, but i have no an idea how to debug it :)

**Comment by ksmandersen** [comment]
@pradel I have this happening too with Next.js

**Comment by aguscha333** [comment]
It was happening to me too. I discovered that it was the `CSSTransition` being cut off and the callback for `onExited` not being called. Which leads to `noScroll.off()`not being called either.
Default transition time is `500` milliseconds, barely noticeable so what I did was set `animationDuration={0}`. If that is not a good quick fix for you there is also the possibility to set `blockScroll` to false. If scroll is never blocked then when you go to another page you don't have the issue.
I went with the solution of setting `animationDuration` to `0` as i mentioned since it was minimal to begin with.
